scripts/export-git-env-vars.sh

```diff
@@ -19,7 +26,9 @@ function find_most_recent_numeric_tag() {
 
 export GITHASH=$(git rev-parse --short=7 HEAD)
 export THIS_COMMIT_TAG=$(git tag --points-at HEAD)
+
 export GITBRANCH=$(git name-rev --name-only $GITHASH | sed "s/remotes\/origin\///g" |  sed 's!/!-!g; s!_!-!g; s!#!-!g' )
+[[ -n "$BRANCH_NAME" ]] && export GITBRANCH=$(echo "$BRANCH_NAME" | sed 's!/!-!g; s!_!-!g; s!#!-!g')
```

Comment:
Suggest not calling `git` at all when `BRANCH_NAME` is provided

Something like the following:

```suggestion
[[ -n "$BRANCH_NAME" ]] && export GITBRANCH=$(echo "$BRANCH_NAME" | sed 's!/!-!g; s!_!-!g; s!#!-!g') || export GITBRANCH=$(git name-rev --name-only $GITHASH | sed "s/remotes\/origin\///g" | sed 's!/!-!g; s!_!-!g; s!#!-!g' )
```